paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -616,7 +607,7 @@ class Paparazzi @JvmOverloads constructor(
     synchronized(this) {
       // https://android.googlesource.com/platform/frameworks/layoutlib/+/d58aa4703369e109b24419548f38b422d5a44738/bridge/src/com/android/layoutlib/bridge/BridgeRenderSession.java#171
       // BridgeRenderSession.executeCallbacks aggressively tears down the main Looper and BridgeContext, so we call the static delegates ourselves.
-      Handler_Delegate.executeCallbacks()
+      Handler_Delegate.executeCallbacks(System_Delegate.nanoTime())
```

Comment:
This is effectifely what `executeCallbacks` used to do internally before, but now it requires the time to be passed in